packages/schemas/src/foundations/schemas.ts

```diff
@@ -23,12 +23,18 @@ export type GeneratedSchema<
   Schema extends SchemaLike<Key>,
   Table extends string = string,
   TableSingular extends string = string,
+  View extends string = string,
 > = Readonly<{
   table: Table;
   tableSingular: TableSingular;
   fields: {
     [key in Key]: string;
   };
+  /**
+   * The view name for the schema.
+   * If not specified, the view name will be the same as the table name.
+   */
+  view: View;
```

Comment:
the purpose of a sole view here looks vague. if we want to use a view to replace the previous table target - then the view should be invisible in code?